src/main/java/com/autotune/analyzer/services/ListStackLayers.java

```diff
@@ -50,7 +50,10 @@ public class ListStackLayers extends HttpServlet {
      * {
      * "experiment_name": "autotune-max-http-throughput",
      * "experiment_id": "94f76772f43339f860e0d5aad8bebc1abf50f461712d4c5d14ea7aada280e8f3",
-     * "objective_function": "request_count",
+     * "objective_function":
+     *  {
+     *  "type": "source"
+     *  },
```

Comment:
In general, we should not be using `source` examples, as a regular user will not be using this. `source` is only used by developers who are adding a new performance profile with a source file attached to it. So general doc should have `expression` examples